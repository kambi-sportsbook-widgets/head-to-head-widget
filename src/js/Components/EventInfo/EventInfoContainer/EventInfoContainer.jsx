import React from 'react';
import styles from './EventInfoContainer.scss';

const EventInfoContainer = ({ children }) => {
   return (<div className={styles.general}>{ children }</div>)
};

EventInfoContainer.propTypes = {
   children: React.PropTypes.object
};

export default EventInfoContainer;
