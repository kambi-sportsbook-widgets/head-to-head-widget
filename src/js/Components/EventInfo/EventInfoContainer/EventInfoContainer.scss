:local(.general) {
   display: flex;
   flex-flow: row nowrap;
   flex-grow: 1;
   justify-content: center;
}
