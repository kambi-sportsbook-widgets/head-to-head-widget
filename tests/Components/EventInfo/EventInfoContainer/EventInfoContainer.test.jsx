/* eslint-env jest */
import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import EventInfoContainer from '../../../../src/js/Components/EventInfo/EventInfoContainer/EventInfoContainer';

let renderer;

describe('EventInfoContainer.render()', () => {

   beforeEach(() => {
      renderer = new ReactShallowRenderer();
   });

   it('renders correctly with children', () => {
      expect(renderer.render(
         <EventInfoContainer>
            <div />
            <div />
         </EventInfoContainer>
      )).toMatchSnapshot();
   });

});
