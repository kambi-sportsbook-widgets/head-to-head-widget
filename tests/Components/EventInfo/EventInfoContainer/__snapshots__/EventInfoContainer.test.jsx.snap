// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventInfoContainer.render() renders correctly with children 1`] = `
<div
  className="general"
>
  <div />
  <div />
</div>
`;
